epregressions/tests/builds/test_base.py

```diff
@@ -58,11 +62,17 @@ def test_verify_without_setting_build_dir(self):
     def test_get_idfs(self):
         temp_idf_dir = tempfile.mkdtemp()
         self.assertSetEqual(set(), self.base_build.get_idfs_in_dir(temp_idf_dir))
-        open(os.path.join(temp_idf_dir, 'file1.idf'), 'w').write('hi')
-        open(os.path.join(temp_idf_dir, 'file2.iQQ'), 'w').write('he')
-        open(os.path.join(temp_idf_dir, 'file3.idf'), 'w').write('ha')
-        open(os.path.join(temp_idf_dir, 'file4.imf'), 'w').write('ha')  # macro
-        open(os.path.join(temp_idf_dir, '_ExternalInterface-actuator.idf'), 'w').write('ha')  # ext interface as FMU
-        open(os.path.join(temp_idf_dir, 'HVAC3ZoneGeometry.imf'), 'w').write('ha')  # macro resource file
+        with open(os.path.join(temp_idf_dir, 'file1.idf'), 'w') as f:
+            f.write('hi')
+        with open(os.path.join(temp_idf_dir, 'file2.iQQ'), 'w') as f:
+            f.write('he')
+        with open(os.path.join(temp_idf_dir, 'file3.idf'), 'w') as f:
+            f.write('ha')
+        with open(os.path.join(temp_idf_dir, 'file4.imf'), 'w') as f:
+            f.write('ha')  # macro
+        with open(os.path.join(temp_idf_dir, '_ExternalInterface-actuator.idf'), 'w') as f:
+            f.write('ha')  # ext interface as FMU
+        with open(os.path.join(temp_idf_dir, 'HVAC3ZoneGeometry.imf'), 'w') as f:
+            f.write('ha')  # macro resource file
```

Comment:
Avoid warning when running unittests `ResourceWarning: unclosed file <_io.TextIOWrapper`

Reply:
I guess could've appended .close() on each of the lines, but this is good.